### DISCUSSION TITLE
sesh + Alfred

### DISCUSSION THREAD

**Discussion by ntawileh** [joshmedeski/sesh#106] [Show and tell]
Alfred workflow for sesh:
    
![sesh-alfred](https://github.com/joshmedeski/sesh/assets/5069918/368c710f-a9ac-40c5-a1e1-bc85baaf3d65)

While I should probably try out Raycast, I haven't yet and I'm still using Alfred!  I put this quick workflow together so I could get the equivalent functionality that the sesh Raycast extension provides.  Sharing it here in case anyone else is interested!

https://github.com/ntawileh/sesh-alfred

**Comment by joshmedeski** [comment]
Nice job! 👏
